app/src/main/java/ar/com/wolox/android/example/ui/home/HomeFragment.kt

```diff
@@ -0,0 +1,17 @@
+package ar.com.wolox.android.example.ui.home
+
+import ar.com.wolox.android.R
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+
+class HomeFragment : WolmoFragment<HomePresenter>() {
+
+    override fun init() { }
+
+    override fun layout(): Int {
+        return R.layout.fragment_home
+    }
+
+    companion object {
+        val instance = HomeFragment()
```

Comment:
Define a function `newInstance()`that returns `HomeFragment()` so we have Java and Kotlin classes coherence